Code/Framework/AzFramework/AzFramework/Physics/HeightfieldProviderBus.h

```diff
@@ -71,11 +71,11 @@ namespace Physics
 
         //! Returns the height field gridsize columns.
         //! @return the size of the grid in the x direction.
-        virtual size_t GetHeightfieldGridColumns() const = 0;
```

Comment:
Probably too late to change this but u64 seems a bit excessive here - that would be enough to do a terrain the size of the solar system at nanometre resolution.

Reply:
That might be the case, I was being conservative here, didn't want to make any assumptions. Although in C++ it's not specified the maximum size of a `size_t`, in general it'll be 64bits on 64-bits systems. I went for the safest approach and use the same size with `AZ::u64`.